fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -4529,6 +4532,42 @@ private Expression getWildWhere(DorisParser.WildWhereContext ctx) {
         }
     }
 
+    @Override
+    public LogicalPlan visitShowColumnStats(ShowColumnStatsContext ctx) {
+        List<String> tableNameParts = visitMultipartIdentifier(ctx.tableName);
+        List<String> colNames = ctx.columnList == null ? ImmutableList.of() : visitIdentifierList(ctx.columnList);
+        PartitionNames partitionNames = parsePartitionSpec(ctx.partitionSpec());
+        boolean isCached = ctx.CACHED() != null;
+
+        return new ShowColumnStatsCommand(new TableNameInfo(tableNameParts), colNames, partitionNames, isCached);
+    }
+
+    private PartitionNames parsePartitionSpec(PartitionSpecContext ctx) {
```

Comment:
there is a visitPartitionSpec method already, please use it instead of creating a new one. Take bellow as example:
```
Pair<Boolean, List<String>> partitionSpec = visitPartitionSpec(ctx.partitions);
PartitionNamesInfo partitionNames = new PartitionNamesInfo(partitionSpec.first, partitionSpec.second);
```